cleanlab/count.py

```diff
@@ -74,6 +75,8 @@ def num_label_issues(
     labels :
       An array of shape ``(N,)`` of noisy labels, i.e. some labels may be erroneous.
       Elements must be in the set 0, 1, ..., K-1, where K is the number of classes.
+      For multi-label classification where each example can belong to multiple classes,
+      refer to documentation for this argument in filter.find_label_issues() for details.
```

Comment:
need to link to the function properly, see development.md for instructions on how to link to other methods in docstrings

Reply:
Fixed: https://github.com/aditya1503/cleanlab/blob/a684583aa0e89de6414172c891b1db305e31ade3/cleanlab/count.py#L79